rpc/cloudagent/cloud/controlplane/moc_cloudagent_controlplane.proto

```diff
@@ -42,6 +42,14 @@ message ControlPlane {
 	Status status = 6;
 	// state stores the last known election status of this ControlPlane
 	ControlPlaneState state = 7;
+	// etcd client port to listen on
```

Comment:
Can you create a separate Object and move these  there

Reply:
@madhanrm @bfjelds 

In that case, I think I would create two:

etcdcluster
-id
-name
-locationName
-cacertificate
-cakey
-status

etcdpeer
-id
-name
-etcdClusterName
-locationName
-fqdn
-clientport
-peerport
-status

Let me know if you have any feedback. Also, is there some script for making most of the copying or is it "pick your favority and copy/replace"? Also, thinking about putting these under cloud, but let me know if you have a preference.

Reply:
Thinking more, perhaps etcd peer could be just an array property of etcdcluster.

Reply:
Decided to keep them split. Let me know if you do not like the approach.

Reply:
I was just asking to create a separate Class and not an entire service.

Reply:
service is also fine, but needs more work

Reply:
Aah, misunderstood. I think I have all the pieces ready now, except for the hierarchy. Let me know if you do not see usage for the service and I can fold it back.

Reply:
@madhanrm, please let me know if you have a preference whether to keep the service or not.

Reply:
Thinking more, if we want to have a two different providers under controlplane for leader election, it does make sense to have these under its own service. So will proceed that way.